internal/app/config/local.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+
+// Description: This file has logic for reading and parsing local config
+package config
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
+)
+
+// ReadLocalDevenv returns DevenvConfig from local filesystem
+func ReadLocalDevenv(path string) (*DevenvConfig, error) {
+	f, err := os.Open(filepath.Join(path, DevenvConfigFileName))
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to read %s", DevenvConfigFileName)
+	}
+	defer f.Close()
+
+	var cfg DevenvConfig
+	if err := yaml.NewDecoder(f).Decode(&cfg); err != nil {
+		return nil, errors.Wrapf(err, "failed to parse %s", DevenvConfigFileName)
+	}
+
+	return &cfg, nil
+}
+
+// ReadServiceName returns service name from local service config
+func ReadServiceName(path string) (string, error) {
+	// read the repository's service.yaml for bootstrap applications
+	b, err := os.ReadFile(filepath.Join(path, ServiceConfigFileName))
+	if err != nil {
+		return "", errors.Wrapf(err, "failed to read %s", ServiceConfigFileName)
+	}
+
+	// conf is a partial of the configuration file for services configured with
+	// bootstrap (stencil).
+	var conf struct {
+		Name string `yaml:"name"`
+	}
+
+	if err = yaml.Unmarshal(b, &conf); err != nil {
```

Comment:
Since I see it right now

```suggestion
	if err := yaml.Unmarshal(b, &conf); err != nil {
```